### PR TITLE
feature: add FR Layout

### PR DESCRIPTION
**Pull request by utix**

### CODE REVIEW

File: data/matrix_layouts/razerdefault18.json [hunk 1]
```diff
@@ -209,5 +209,111 @@
             {"label": "\ud83e\udc3b", "matrix": [5, 16]},
             {"label": "\ud83e\udc3a", "matrix": [5, 17]}
         ]
+    },
+        "French": {
```
Comment (z3ntu) [inline data/matrix_layouts/razerdefault18.json:213]: Please remove 4 spaces in this row so it's aligned with the closing bracket above

**Comment by z3ntu** [comment]
Thanks! I'll check it at runtime in the next days. Until then added one small comment, I could also fix it up while merging but would be nice if you could do that.

File: data/matrix_layouts/razerdefault18.json (hunk 1 above)
Comment (utix) [reply data/matrix_layouts/razerdefault18.json]: My bad fixed

**Comment by z3ntu** [comment]
Visually looks nice! Just adjusted some other tiny things, thanks for the patch!

**Review by z3ntu — APPROVED** [review]

**Pull request merged**